tools/verify-origami-json/index.js

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env node
+
+import process from 'process'
+import isCI from 'is-ci'
+import { EOL } from 'node:os'
+import { readFile } from 'node:fs/promises'
+import path from 'node:path'
+import { fileExists } from 'origami-tools-helpers'
+
+// https://origami.ft.com/docs/manifests/origami-json/#origamitype
+// "component": A front-end component that follows the component specification
+// "imageset": A set of images that have an alias on the Origami Image Service
+// "service": An HTTP service that follows the service specification
+// "cli": A command line tool
+// "library": A library that is not a front-end component
+// "website": Origami websites that aren’t intended to be services
+// "config": Projects that are configuration for other projects
+// "example": Example and boilerplate projects
+// "meta": Repository-only projects that relate to how Origami works
+// null: An Origami project that does not fit any of the named categories
+
+const cwd = process.argv[2]
```

Comment:
```suggestion
const cwd = process.argv[2] || process.cwd()
```

let's fall back to the current directory so people can run it like `npm exec verify-origami-json` from inside a component